src/uct/api/tl.h

```diff
@@ -71,6 +67,14 @@ typedef struct uct_iface_ops {
     int          (*iface_is_reachable)(uct_iface_h iface,
                                        const uct_device_addr_t *addr);
 
+    void         (*ep_destroy)(uct_ep_h ep);
```

Comment:
but `uct_base_iface_set_error()` is currently not doing anything - so we should postpone the change (and the comment) to the implementation PR